pkg/lib/operation/project/local/template/use/operation.go

```diff
@@ -273,7 +299,12 @@ func (p *TemplatePlan) Invoke(ctx context.Context) (*Result, error) {
 		logger.Warn(ctx, `Push operation is only possible when project is valid.`)
 	}
 
-	result := &Result{InstanceID: p.options.InstanceID}
+	result := p.result
+	if result == nil {
+		result = &Result{InstanceID: p.options.InstanceID}
+	} else {
+		result.InstanceID = p.options.InstanceID
+	}
```

Comment:
What is this change for?

Reply:
It could be omited as the template use creates `InstanceID` which UI uses. I wanted to add `ConfigID` of the template created juts for info but it will not be used.

Reply:
I don't get it. How is InstanceID related to ConfigID?

Reply:
It's not related, it is in returned values.

Reply:
I checked the code again, I think I understand now. Let's keep this change. :+1: